test/integration/standard_filter_test.rb

```diff
@@ -467,10 +471,38 @@ def test_sort_calls_to_liquid
   end
 
   def test_map_over_proc
-    drop  = TestDrop.new(value: "testfoo")
+    drop  = TestDrop.new(value: "123")
     p     = proc { drop }
     templ = '{{ procs | map: "value" }}'
-    assert_template_result("testfoo", templ, "procs" => [p])
+    assert_template_result("123", templ, "procs" => [p])
+  end
+
+  def test_join_over_proc
+    drop  = TestDrop.new(value: "123")
+    p     = proc { drop }
+    templ = '{{ procs | join }}'
+    assert_template_result('TestDrop(value:123)', templ, "procs" => [p])
+  end
+
+  def test_sort_over_proc
+    drop  = TestDrop.new(value: "123")
+    p     = proc { drop }
+    templ = '{{ procs | sort: "value" }}'
+    assert_template_result("TestDrop(value:123)", templ, "procs" => [p])
+  end
+
+  def test_where_over_proc
+    drop  = TestDrop.new(value: "123")
+    p     = proc { drop }
+    templ = '{{ procs | where: "value", "123" }}'
+    assert_template_result("TestDrop(value:123)", templ, "procs" => [p])
+  end
+
+  def test_uniq_over_proc
+    drop  = TestDrop.new(value: "123")
+    p     = proc { drop }
+    templ = '{{ procs | uniq }}'
+    assert_template_result("TestDrop(value:123)", templ, "procs" => [p])
```

Comment:
These tests are to cover the new behavior for Proc over filters where historically they would have raised on missing method `to_liquid`